modules/sdp/SDP.js

```diff
@@ -558,53 +554,53 @@ SDP.prototype.fromJingle = function(jingle) {
 
 // translate a jingle content element into an an SDP media part
 SDP.prototype.jingle2media = function(content) {
-    const desc = content.find('>description');
-    const transport = content.find('>transport[xmlns="urn:xmpp:jingle:transports:ice-udp:1"]');
+    const desc = content.querySelector(':scope >description');
+    const transport = content.querySelector(':scope >transport[*|xmlns="urn:xmpp:jingle:transports:ice-udp:1"]');
     let sdp = '';
-    const sctp = transport.find(
-        '>sctpmap[xmlns="urn:xmpp:jingle:transports:dtls-sctp:1"]');
+    const sctp = transport.querySelector(':scope >sctpmap[*|xmlns="urn:xmpp:jingle:transports:dtls-sctp:1"]');
 
-    const media = { media: desc.attr('media') };
+    const media = { media: desc.getAttribute('media') };
```

Comment:
A jQuery object will return undefined if `attr` is called, while calling `getAttribute` on an element which was not found will crash. So to mimic the behavior we would need `someElement.?getAttribute` if I am not mistaken.

Reply:
I would try it like that because everything should be set anyway, if not we can find more errors in the jitsi lib.